c++/cmake/CapnProtoMacros.cmake

```diff
@@ -0,0 +1,116 @@
+# CAPNP_GENERATE_CPP ===========================================================
+#
+# Example usage:
+#   find_package(CapnProto)
+#   capnp_generate_cpp(CAPNP_SRCS CAPNP_HDRS schema.capnp)
+#   include_directories(${CMAKE_CURRENT_BINARY_DIR})
+#   add_executable(foo main.cpp ${CAPNP_SRCS})
+#   target_link_libraries(foo CapnProto::capnp)
+#
+#  If you are using not using the RPC features you can remove
```

Comment:
Invert the sentence to match the example: "If you need the RPC features, add 'CapnProto::capnp-rpc' to the target_link_libraries() call."